examples/baselines/2dhistos/vars.json

```diff
@@ -0,0 +1,7 @@
+{
+    "nevents" : [2000000],
+    "run_params" : ["3pt74"],
```

Comment:
You only are using nevents and run_number from this currently. I would suggest not giving nevents so it just processes all of them, and extracting the run number from the filename. That way you can just eliminate this file.